addons/base/models.py

```diff
@@ -108,7 +108,7 @@ def nodes_authorized(self):
         model = self.config.node_settings
         if not model:
             return []
-        return [obj.owner for obj in model.objects.filter(user_settings=self, owner__is_deleted=False)]
+        return [obj for obj in model.objects.filter(user_settings=self, owner__is_deleted=False).select_related('owner')]
```

Comment:
Hmm. You're right, but this somehow doesn't break the links to nodes on `/settings/addons`, which is what I thought this populated.